aap_chatbot/src/AnsibleChatbot/AnsibleChatbot.tsx

```diff
@@ -34,32 +32,11 @@ import {
   ChatbotToggle,
   Conversation,
 } from "@patternfly/chatbot";
-import {
-  CHAT_HISTORY_HEADER,
-  FOOTNOTE_DESCRIPTION,
-  FOOTNOTE_LABEL,
-  FOOTNOTE_TITLE,
-} from "../Constants";
+import { CHAT_HISTORY_HEADER, FOOTNOTE_LABEL } from "../Constants";
 import { SystemPromptModal } from "../SystemPromptModal/SystemPromptModal";
 
 const footnoteProps: ChatbotFootnoteProps = {
   label: FOOTNOTE_LABEL,
-  popover: {
```

Comment:
if we're removing the footnote popover, we can remove this footnote popover [useEffect](https://github.com/ansible/ansible-ai-connect-service/blob/main/aap_chatbot/src/AnsibleChatbot/AnsibleChatbot.tsx#L149-L179) logic as well

Reply:
Also the "Basic chatbot interaction" test case in App.text.tsx attempts to open the popover. That part needs to be removed.

Reply:
good catch, let me check

Reply:
fixed, thanks guys!